aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -207,6 +240,13 @@ module aptos_framework::voting {
         let proposal_id = voting_forum.next_proposal_id;
         voting_forum.next_proposal_id = voting_forum.next_proposal_id + 1;
 
+        // Add a flag to indicate if this proposal is single-step or multi-step.
+        simple_map::add(&mut metadata, utf8(IS_MULTI_STEP_PROPOSAL_KEY), to_bytes(&is_multi_step_proposal));
+        if (is_multi_step_proposal) {
+            // Add a flag to indicate if a multi-step proposal is in execution. If yes, we will disable voting.
+            simple_map::add(&mut metadata, utf8(IS_MULTI_STEP_PROPOSAL_IN_EXECUTION_KEY), to_bytes(&false));
+        };
```

Comment:
Just for defensive programming, can we remove IS_MULTI_STEP_PROPOSAL_IN_EXECUTION_KEY if it's set in the metadata by client but is_multi_step_proposal is false? This would keep things consistent later on